SwiftyBluetooth/Source/CentralProxy.swift

```diff
@@ -200,6 +203,56 @@ extension CentralProxy {
         }
     }
     
+    func connect(peripheralUUID: UUID, serviceUUIDs: [CBUUID], timeout: TimeInterval, _ callback: @escaping ConnectPeripheralCallback) {
```

Comment:
No worries, will just maintain an alternate fork of this repo then--I don't think it is possible to achieve the utility we need with extensions with the way that the scoping is currently set up, without calling underlying functions like `retrievePeripherals()` multiple times.